src/App/AppContainer.js

```diff
@@ -118,6 +126,7 @@ class AppContainer extends Component {
                         <div>
                             <Route exact path='/:resource/admin' render={(props) => <AdminPage currentPosition={this.state.position} />} />
                             <Route exact path='/:resource/' render={(props) => <MapPage currentPosition={this.state.position} />} />
+							<Route path='/:resource/' render={NotFoundPage} />
```

Comment:
![404](https://user-images.githubusercontent.com/33295600/59519800-ac643e00-8e96-11e9-8364-fe181cfd80a4.png)

looks good, just take out this line so the 404 page isn't rendered on every page, and it's perfect.